packages/device-connector/src/arduino-cli/client.ts

```diff
@@ -227,12 +265,12 @@ export class RPCClient {
         stream.destroy()
       })
 
-      stream.on('status', (status) => {
-        return status.code === 0 ? resolve(true) : reject(new Error(status.details))
+      stream.on('status', (status: StatusObject) => {
+        return status.code === 0 ? resolve(true) : reject(status)
```

Comment:
reject with error

Reply:
StatusObjects are part of the errors in grpc. The newest commit handles these to properly log the status error details